### DISCUSSION TITLE
im new to github,how do i know where to post?

### DISCUSSION THREAD

**Discussion by ZeusSwerveBoi** [community/community#36102] [Other Feature Feedback, Questions, & Ideas]
hello there,
i made a github account recently cuz i was having issues with process hacker and it took me to : https://github.com/winsiderss
/
[systeminformer](https://github.com/winsiderss/systeminformer)
to make a issue. 
but how do i know what is the link for other programs i need help with? this is confusing to me cuz idk what a "repositories" is. "winsiderss
/
[systeminformer]" doesn't tell me its for process hacker. on other forum sites like se7ensins i just go to a thread like gta5 and find gta5mods,support,etc. like the naming is simple. github is different.

**Comment by DSDmark** [comment]
Hey @ZeusSwerveBoi,

So, I understand a little bit. You want you to help that [systeminformer](https://github.com/winsiderss/systeminformer) **repository** because 
you face some issues, but you don't know how to help that **repository**.

> Why that way **repository** you can imagine as a folder that contains software or package or maybe a tool it's depending upon the project
So, Repositories in git or GitHub contain a collection of files of various different versions of a Project. It's nothing more.

Back to the topic. For help with that **systeminformer** **repository** just you need to submit the **issue** there are already several issues submitted on that repo. For creating a new issue, click on that **issue** icon.

- ![Screenshot from 2022-10-14 10-54-11](https://user-images.githubusercontent.com/85744816/195768748-df736252-c4fd-434d-8174-1120e8cf18ad.png)

### then create a new issue like this:- 

- ![Screenshot from 2022-10-14 10-55-28](https://user-images.githubusercontent.com/85744816/195769030-96027915-c71c-48c7-8615-34023e262ab5.png)

### And then click on new issue and select your type of issue.
- ![Screenshot from 2022-10-14 10-58-28](https://user-images.githubusercontent.com/85744816/195769400-e71b2e3a-c913-4890-a3de-7879d5b80879.png)

### And tell what your issue is by filling the fields

**Reply by DSDmark** [reply]
## Hey also try to add label when you ask any question on that community it's help you to and other developer to help you. If you need help, then select **help label**.

![Screenshot from 2022-10-14 11-06-58](https://user-images.githubusercontent.com/85744816/195770436-829d48b7-e64c-4805-a6f3-1b2806747fad.png)

**Reply by ZeusSwerveBoi** [reply]
thank you for the reply mark.  but i had a little trouble understanding you're english no offense. im still a little confused on this subject, how do i know which repository to go to for a certian project/program?

**Reply by DSDmark** [reply]
Sorry for that, i m not a native speaker. So, About the project as I understood, **process hacker** is a tool for using debugging software and detect malware. 

> for you understanding the **systeminformer** it is the official repository for **process hacker** .so, if you want anything go there and submit your problem, or you can discuss you problem.

For discussing the topic about **process hacker** go **systeminformer** and create a discussion like that :- 

### click on discussions and create a new discussion on the official **process hacker** repo.

- ![Screenshot from 2022-10-14 12-19-08](https://user-images.githubusercontent.com/85744816/195781253-b38a119d-ef23-419e-b93d-56ca45995b42.png)

---

- ![Screenshot from 2022-10-14 12-23-40](https://user-images.githubusercontent.com/85744816/195781751-9a86463a-dcc2-4370-b172-d81f6bd10472.png)

**Reply by ZeusSwerveBoi** [reply]
my bad for the late reply i been busy, i just want to reclarify what i typed. "how do i know which repository to go to for a certian project/program?" so i already know how to post issues and discussions, i just would like to know how to find out what is the correct repository for a program or project? like for process hacker i found the repository on : https://processhacker.sourceforge.io

**Reply by DSDmark** [reply]
Ok then, I hope you get your answer. If you get your answer, then you should close that question by click **accept answer** button, or you can write the answer and **accept** own answer.
